src/main.js

```diff
@@ -25,6 +26,22 @@ ideaCardContainer.addEventListener("click", determineIdeaCardEvent);
 window.addEventListener("load", onload);
 
 // FUNCTIONS:
+function findIdeas(event) {
+  var foundIdeas = [];
+  var inputByUser = event.target.value.toLowerCase()
+  for (var i = 0; i < userIdeas.length; i++){
+    var lowerCaseTitle = userIdeas[i].title.toLowerCase();
+    var lowerCaseBody = userIdeas[i].body.toLowerCase();
+    if (lowerCaseTitle.includes(inputByUser) || lowerCaseBody.includes(inputByUser)){
+      foundIdeas.push(userIdeas[i])
+      displayUserCards(foundIdeas)
+    }
+    if (foundIdeas.length === 0){
```

Comment:
This search functionality looks great! The function seems a bit long though. Is there a way to refactor a little?